From: chko <hkominos>
Date: Tue, 21 Mar 2017 14:51:22 +0100
Subject: [PATCH] Fixed typo in net-check Closes-Bug:#1674724

Change-Id: Ibf35e5ffb1411b998f9ad250b7744c447e16fa71
Signed-off-by: Charalampos Kominos <hkominos@gmail.com>
(cherry picked from commit e5debec9fe2d8f1a99521594ca3bb10de3bb6d13)
---
 network_checker/net_check/api.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/network_checker/net_check/api.py b/network_checker/net_check/api.py
index defdde0..a4155db 100755
--- a/network_checker/net_check/api.py
+++ b/network_checker/net_check/api.py
@@ -199,7 +199,7 @@ class Actor(object):
         if state.lower() == 'up':
             self.logger.debug('Inteface %s is up', check_iface)
             return True
-        elif state.lower == 'unknown':
+        elif state.lower() == 'unknown':
             self.logger.debug('Inteface %s state is unknown, using ethtool',
                               check_iface)
             ethtool_output = self._execute(['ethtool', check_iface])
